### PR TITLE
Patch/minor tweaks

### PR DESCRIPTION
**Pull request by jetzlstorfer**

### CODE REVIEW

File: COMMUNITY_MEMBERSHIP.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Members are continuously active contributors in the community. They can have iss
   - Make sure that the list of contributions included is representative of your
     work on the project.
 - Have your sponsoring reviewers reply confirmation of sponsorship: `+1`
-- Once your sponsors have responded, your request will be reviewed by the
+- TO BE CLARIFIED: Once your sponsors have responded, your request will be reviewed by the
```
Comment (jetzlstorfer) [inline COMMUNITY_MEMBERSHIP.md:62]: do we need the GC here? where is the GC defined?

File: COMMUNITY_MEMBERSHIP.md [hunk 2]
```diff
@@ -134,7 +134,7 @@ The process for becoming a maintainer should be defined in the project or subpro
     meetings)
   - Identifying subtle or complex issues in designs and implementation PRs
 - Directly contributed to the subproject through implementation and / or review
-- Aligning with the overall project goals, specifications, and design principles defined by Technical Committee (TC). Bringing general questions and requests to the discussions as part of specifications project.
+- Aligning with the overall project goals, specifications, and design principles defined by Technical Committee (TC) (NEEDS CLARIFICATION). Bringing general questions and requests to the discussions as part of specifications project.
```
Comment (jetzlstorfer) [inline COMMUNITY_MEMBERSHIP.md:137]: similar to the GC, where is the TC defined and do we need it?

**Review by jetzlstorfer — COMMENTED** [review]
FYI @johannes-b @christian-kreuzberger-dtx

**Comment by christian-kreuzberger-dtx** [comment]
IMHO we don't need GC nor TC.

**Comment by johannes-b** [comment]
I agree with Christian and don`t see a need for GC nor TC yet.

File: COMMUNITY_MEMBERSHIP.md (hunk 2 above)
Comment (johannes-b) [reply COMMUNITY_MEMBERSHIP.md:137]: Please see the comments below.

**Comment by jetzlstorfer** [comment]
remove the GC and TC and reworded it to "maintainers" in both cases

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**